src/tpu.rs

```diff
@@ -56,7 +56,7 @@ impl Tpu {
         bank: &Arc<Bank>,
         crdt: &Arc<RwLock<Crdt>>,
         tick_duration: Option<Duration>,
-        transactions_socket: UdpSocket,
+        transactions_socket: Vec<Arc<UdpSocket>>,
```

Comment:
To make tests a little easier to write, can you leave that without the Arc? So `Vec<UdpSocket>`.

And then, in FetchStage:

```rust
let tx_sockets = transactions_socket.into_iter().map(Arc::new).collect();
```

Reply:
Done.